flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaWriterITCase.java

```diff
@@ -148,6 +148,15 @@ public void testIncreasingRecordBasedCounters() throws Exception {
             assertThat(numRecordsOutErrors.getCount()).isEqualTo(0);
             assertThat(numRecordsSendErrors.getCount()).isEqualTo(0);
 
+            // elements that cannot be serialized should be silently skipped
```

Comment:
minor: we can improve the note, because not all serializers will return `null` when the element cannot be serialized, not all `null` are due to cannot be serialized . 
What we can ensure is only that the `KafkaSinkWriter` will silently skip `null` that returned by serializers

Reply:
Agreed. Maybe something along these lines?

```suggestion
            // elements for which the serializer returns null should be silently skipped
```

Reply:
> Agreed. Maybe something along these lines?

Yes, this is better.